src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user?: User,
```

Comment:
let's improve our types. since user after your map always exists in todo let's make it like this
```suggestion
  user: User || null,
```